code/scripting/lua/LuaTable.h

```diff
@@ -19,7 +19,7 @@ class LuaTable;
  * be in the exact same state when you call the next method of this class.
  */
 class LuaTableIterator {
-	lua_State* _L = nullptr;
+	lua_State* L_State = nullptr;
```

Comment:
Ah. I was reluctant to do that originally because leading underscores are theoretically reserved for the standard headers...